library/control/src/modules/InstError.rb

```diff
@@ -45,26 +45,19 @@ def main
     end
 
     def SaveLogs
-      cmd = Convert.to_map(
-        WFM.Execute(path(".local.bash_output"), "echo ${HOME}")
-      )
-      homedir = "/"
+      env_home = ENV["HOME"]
 
-      if Ops.get_integer(cmd, "exit", -1) == 0
-        homedir = Ops.get(
-          Builtins.splitstring(Ops.get_string(cmd, "stdout", "/"), "\n"),
-          0,
-          "/"
-        )
-        homedir = "/" if homedir == ""
+      homedir = if env_home && !env_home.empty?
+        env_home
       else
         Builtins.y2warning(
           "Unable to find out home dir: %1, using %2",
           cmd,
```

Comment:
`cmd` has been deleted

Reply:
More precisely, the variable now longer exists before the `else` block when it is still referenced, so we will get `nil` there which may or may not result in a nil.NilClass exception (depending how smart our y2warning is).

Reply:
well, cmd cause method not found exception. for homedir it will be nil, so no problem for y2warning.

JFYI how it can be verified in irb:
```
irb(main):001:0> test = if false; puts "no"; else puts test.inspect; end
nil
=> nil
```